src/Auxiliary/Aux_FindWeightedAverageCenter.cpp

```diff
@@ -209,7 +209,7 @@ void Aux_FindWeightedAverageCenter( double WeightedAverageCenter[], const double
                                   MinDens_No, MinPres_No, MinTemp_No, MinEntr_No, DE_Consistency_No );
             }
 
-#           pragma omp parallel for reduction ( +:W_ThisRank, WX_ThisRank, WY_ThisRank, WZ_ThisRank ) schedule( runtime )
+#           pragma omp parallel for reduction ( +:W_ThisRank, WX_ThisRank, WY_ThisRank, WZ_ThisRank ) schedule( static )
```

Comment:
Add the comment "//       use the "static" schedule for reproducibility"

Reply:
Done!